examples/diagnostics/compare_diags.py

```diff
@@ -134,6 +157,13 @@ def print_totals_and_diffs(config, refdata, devdata, varlist):
     filename = config["options"]["totals_and_diffs"]["filename"]
     do_screen = config["options"]["totals_and_diffs"]["print_to_screen"]
 
+    # Determine if we will print percent or absolute differences
+    do_percent_diff = False
+    if "percent" in config["options"]["totals_and_diffs"]["diff_type"] or \
+       "pctdiff" in config["options"]["totals_and_diffs"]["diff_type"] or \
+       "%" in config["options"]["totals_and_diffs"]["diff_type"]:
```

Comment:
If you wanted to try out list comprehension you could simplify this conditional using `any`:
```
matches = ["percent", "pctdiff", "%"]
if any(x in config["options"]["totals_and_diffs"]["diff_type"] for x in matches):
```